src/insert/spacefinder/article-body-adverts.ts

```diff
@@ -232,12 +232,12 @@ const addDesktopRightRailAds = (): Promise<boolean> => {
 	const largestSizeForSlot = adSizes.halfPage.height;
 	const rules: SpacefinderRules = {
 		bodySelector: articleBodySelector,
-		candidateSelector: ' > p',
+		candidateSelector: ':scope > p, [data-spacefinder-role="nested"] > p',
```

Comment:
Do we not need scope here?

```suggestion
		candidateSelector: ':scope > p, :scope [data-spacefinder-role="nested"] > p',
```

Reply:
Not necessarily, `[data-spacefinder-role="nested"] > p` seems specific enough? happy to prefix all the nested selectors with scope also though